package.json

```
@@ -40,7 +40,7 @@
   },
   "homepage": "https://github.com/iterative/dvc.org#readme",
   "engines": {
-    "node": "^18.x"
+    "node": ">=18.x <=19.x"
```

Note that this is running fine on Node v20 as well. Even if we are running in v18, it might be better to relax this constraint here, unless someone complains. 
```suggestion
    "node": ">=18.x <=20.x"
```